models/src/main/java/io/sphere/sdk/reviews/queries/ReviewsEndpoint.java

```diff
@@ -0,0 +1,8 @@
+package io.sphere.sdk.reviews.queries;
```

Comment:
Yes, it is intended to be used that way. The endpoint classes decode the type references for the JSON deserialization and the path for the endpoint. Since I don't want them exposed to the public API, they need to be package scope in two packages.